cleverhans/utils_tf.py

```diff
@@ -516,3 +516,17 @@ def get_available_gpus():
   """
   local_device_protos = device_lib.list_local_devices()
   return [x.name for x in local_device_protos if x.device_type == 'GPU']
+
+
+class HeReLuNormalInitializer(tf.initializers.random_normal):
```

Comment:
This is also in cleverhans_tutorials.tutorial_models, right? We should remove that copy and turn it into an import

Reply:
+1